visualization/app/codeCharta/ui/screenshotButton/screenshotButton.component.ts

```diff
@@ -88,12 +92,38 @@ export class ScreenshotButtonComponent {
 		renderer.render(this.threeSceneService.scene, this.threeCameraService.camera)
 	}
 
-	private buildScreenShotCanvas(renderer: WebGLRenderer): HTMLCanvasElement {
+	private async buildScreenShotCanvas(renderer: WebGLRenderer): Promise<HTMLCanvasElement> {
 		renderer.setPixelRatio(window.devicePixelRatio)
 		renderer.setClearColor(new Color(0, 0, 0), 0)
 		renderer.render(this.threeSceneService.scene, this.threeCameraService.camera)
-		const canvas = renderer.domElement
-		return canvas
+
+		const tagsNamesToIgnore = new Set([
```

Comment:
making a screenshot when one of the debug canvases that are displayed in development mode is shown, they are included in the screenshot. Not sure if it is important to exclude them here as its nothing the average user will ever see but i thought i should include it here because i noticed it.